CONTRIBUTING.md

```diff
@@ -369,3 +370,75 @@ npm --version
 If your installation was successful you will get the versions of node and npm that were installed.
 
 </details>
+
+## PostCSS
+In our repo, we use PostCSS plug-ins to help simplify how we write our CSS. PostCSS is included in our webpack configuration, so there are no additional steps necessary to leverage these plug-ins.
+
+### What is Post CSS? 
+"PostCSS is a tool for transforming styles with JS plugins. These plugins can lint your CSS, support variables and mixins, transpile future CSS syntax, inline images, and more." - [PostCSS Repo](https://github.com/postcss/postcss)
+
+### Modules in Use
+
+[autoprefixer](https://github.com/postcss/autoprefixer): used to parse vendor prefixes for certain CSS selectors 
+([What is a vendor prefix?](https://developer.mozilla.org/en-US/docs/Glossary/Vendor_Prefix)). In our repo, you will not have to include vendor prefixes when you create a non-standard CSS selector. 
+
+**Example:**
```

Comment:
Actually this is an example of `postcss-custom-properties`


Autoprefixer would be like... not needing to do `-webkit-box-sizing` in addition to `box-sizing` rules.